packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -265,6 +265,26 @@ export interface ISemanticColors {
    */
   listItemBackgroundCheckedHovered: string;
 
+  /**
+   * The color of a link.
+   */
+  linkText: string;
```

Comment:
the "text" on these names is redundant, I'd prefer to remove them just to save a few bytes. just calling it "link" and "linkHovered" is clear enough

Reply:
I was debating that, too. The documentation says to use [category name][element name] but for a simple component like this, there isn't much need to include the element name. I'll simplify these.